File: CS/Scripts/Syntax/CaseInsensitivePatternDefinition.cs
using System.Collections.Generic;

namespace ScriptHelp.Scripts.Syntax
{
    public class CaseInsensitivePatternDefinition : PatternDefinition
    {
        public CaseInsensitivePatternDefinition(IEnumerable<string> tokens)
            : base(false, tokens)
        {
        }

        public CaseInsensitivePatternDefinition(params string[] tokens)
            : base(false, tokens)
        {
        }
    }
}

File: VB/Scripts/Syntax/CaseInsensitivePatternDefinition.vb
Imports System.Collections.Generic

Namespace Scripts
    Namespace Syntax

        Public Class CaseInsensitivePatternDefinition

            Inherits PatternDefinition

            Public Sub New(tokens As IEnumerable(Of String))
                'MyBase.New(False, tokens)
            End Sub

            Public Sub New(ParamArray tokens As String())
                'MyBase.New(False, tokens)
            End Sub

        End Class

    End Namespace
End Namespace